cluster/addons/fluentd-elasticsearch/es-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta1
+kind: Pod
```

Comment:
Yes, I had intended to change this to have a replication controller (still with one controlled pod). I'll make the change.